plugins/scanners/dos/src/main/kotlin/DosPackageScannerWrapper.kt

```diff
@@ -74,8 +74,8 @@ import org.ossreviewtoolkit.utils.ort.runBlocking
         "stores / reuses scan results on a per-file basis and thus uses its own scan storage.",
     factory = ScannerWrapperFactory::class
 )
-class DosScanner(
-    override val descriptor: PluginDescriptor = DosScannerFactory.descriptor,
+class DosPackageScannerWrapper(
```

Comment:
This should be `DosScannerWrapper` because the base class for scanner plugins is `ScannerWrapper`.